### ISSUE TITLE
Add f base64: support

### ISSUE THREAD

**Issue by radare** [radare/radare2#14324]
We should support base64 namings in flag commands too, not just in CC/CL, the realname must be kept as is, but the flag name must be filtered

**Comment by XVilka** [comment]
I think it should be something like `pj` API, but in reverse, and for base64.

**Comment by ret2libc** [comment]
I think it makes sense to do it only after the new command parser is in place (still far from it). At that point we could convert any argument that starts with base64:

**Comment by radare** [comment]
yes i know, but i wanted to discuss about the topic, otherwise we wont even care about it when designingg the new command parser

**Comment by XVilka** [comment]
@ret2libc probably also a good time to add this too.

**Comment by ret2libc** [comment]
Not yet imo. It's better to wait until more commands are converted to argv style, otherwise it will be too much inconsistent.
